terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -1 +1,31 @@
-# placeholder file
\ No newline at end of file
+environment = "staging"
+aws_profile = "staging-eu-west-2"
+
+# scaling configs
+desired_task_count = 2 # use multi instance in staging
+
+# service configs
+api_local_url = "https://private-api-staging.company-information.service.gov.uk"
+cdn_host = "d6nh3dxv55e16.cloudfront.net"
+chs_url = "https://staging.company-information.service.gov.uk"
+cookie_domain = "company-information.service.gov.uk"
+piwik_site_id = "15"
+piwik_url = "https://matomo.platform.aws.chdev.org/"
+session_create = "off"
+permission_name_download = "/admin/extensions-download"
+permission_name_view = "/admin/extensions-view"
+maximum_extension_requests_per_day = "50"
+feature_request_count = "false"
+max_file_size_bytes = "4194304"
+show_service_unavailable_page = "off"
+too_soon_days_before_due_date = "60"
+max_extension_period_in_months = "12"
+log_level = "debug"
+cookie_name = "__SID"
+default_session_expiration = "3600"
+
+
+# TODO Needed? Present in Mesos Staging config but not CiDev config, variable not defined in the terraform but there's
```

Comment:
The only outstanding issue now (at least from my perspective). I don't think this is needed but what do others think?

Reply:
doesn't look like it is used so can go without

Reply:
Removed, so resolving.